src/mlpack/methods/bias_svd/bias_svd_function.hpp

```diff
@@ -139,6 +139,8 @@ class BiasSVDFunction
 } // namespace svd
 } // namespace mlpack
 
+// \cond DO_NOT_DOCUMENT
```

Comment:
Doxygen didn't know what to do with the ensmallen namespace, so I told it not to document. This can safely be reverted if we don't mind the warning.

Reply:
I think it's also possible to just use a single-character `/*` in the opening of the comment... do you want to try that and see if it works?  I think Doxygen only parses comments that start with `/**`.

Reply:
That would've been so cool if it had worked, unfortunately no such luck:
```
Searching for member function documentation...
/home/ryan/Documents/mlpack/src/mlpack/methods/bias_svd/bias_svd_function.hpp:153: error: no matching class member found for 
  template <>
  template <>
  double Optimize(mlpack::svd::BiasSVDFunction< arma::mat > &function, arma::mat &parameters)
 (warning treated as error, aborting now)
Exiting...
CMakeFiles/doc.dir/build.make:57: recipe for target 'CMakeFiles/doc' failed
make[3]: *** [CMakeFiles/doc] Error 1
```
Even if it did, I'd expect someone to add the missing * back, thinking it to be a mistake.
Maybe we should add a comment describing why we're asking doxygen to skip this section too?

Reply:
Ahh, yeah, I guess you're right.  I think adding a comment would be nice too. :+1: I'd consider the clearer name `NO_DOXYGEN` instead of `DO_NOT_DOCUMENT` (to make it clear that the condition is specifically for doxygen, in case someone doesn't recognize the `\cond`); but, up to you. :+1: